nemo_rl/algorithms/grpo.py

```diff
@@ -286,31 +286,23 @@ def setup(
 def refit_policy_generation(
     policy: ColocatablePolicyInterface,
     policy_generation: GenerationInterface,
-    refit_buffer_size_gb: int,  # GB
+    refit_buffer_size_gb: Optional[int] = None,
```

Comment:
Can you prefix this kwarg with an underscore to discourage users from setting this since it's primarily used for testing?
```suggestion
    _refit_buffer_size_gb: Optional[int] = None,
```